src/frontend/src/flows/login/unknownAnchor.ts

```diff
@@ -145,7 +145,7 @@ const initLinkDevice = (connection: Connection) => {
       "registerUserNumber"
     ) as HTMLInputElement;
 
-    const userNumber = parseUserNumber(userNumberInput.value);
-    await addRemoteDevice(userNumber, connection);
+    const userNumber = parseUserNumber(userNumberInput.value) ?? undefined;
+    await addRemoteDevice(connection, userNumber);
```

Comment:
note: for these kind of stuffs I find object params for function more reliable `addRemoteDevice({connection, userNumber})` - order of the params becomes irrelevant

Reply:
agreed, but the rest of the codebase doesn't do that and I'd rather stay consistent